src/components/Card/Card.jsx

```diff
@@ -14,7 +16,28 @@ const cardTypes = {
 const borderTypes = { DASHED: 'dashed' };
 export const borderColors = { RED: 'red' };
 
-class Card extends React.Component {
+type Props = {|
+    title: string,
```

Comment:
I think our convention is 2 spaces... speaking of which, we need prettier here :/